integrations/mongodb_atlas/src/haystack_integrations/document_stores/mongodb_atlas/filters.py

```diff
@@ -0,0 +1,9 @@
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def haystack_filters_to_mongo(_):
+    # TODO
+    logger.warning("Filtering not yet implemented for MongoDBAtlasDocumentStore")
```

Comment:
Wouldn't it make sense to raise an error here instead? Silently failing (even with a warning) when the user was expecting otherwise is something we should avoid.

Reply:
Right, good point!